cmd/experimental/kueue-viz/INSTALL.md

```diff
@@ -0,0 +1,107 @@
+# Installation with helm
```

Comment:
If we are planning on promoting this out of experimental I think adding a page in our site for this would be useful.

I don’t know if we want to use the readme for this info for production.

Reply:
Sure, this is only for preparation of that.
Let's follow the proper documentation in a following PR.